asv_setup/CMakeLists.txt

```diff
@@ -23,4 +23,10 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
```

Comment:
line 4-24 is  unnecessary for this package and should be removed.

Reply:
Colcon won't build the package without them though? 
Unknown CMake command "ament_package"

Reply:
ah, ok. then just leave it as is

Reply:
Also, can we rename "tf.launch.yaml" to "tf_launch.yaml"?
I think pc.launch.yaml should be deleted as well.

Reply:
pc.launch.yaml will be taken care of when merging.

Reply:
The if(BUILD_TESTING) can be removed.
The convention is to have .launch.yaml so just leaving it as is is fine :)

Reply:
Also remove the comment
# uncomment the following section in order to fill in
# further dependencies manually.
# find_package(<dependency> REQUIRED)